### DISCUSSION TITLE
send all traffic to another tunnel

### DISCUSSION THREAD

**Discussion by zavataafnan** [strongswan/strongswan#1465] [Q&A]
Hi all,

I set up IPSEC and IKEv2 VPN simply and quickly and thanks.
I have another tun0 and I want that IPSEC and IKEv2 use tun0 instead of eth0. is that possible to change table rules, so we achieve this?

Best Regards,
Mostafa

**Comment by tobiasbrunner** [comment]
Not sure what exactly you are trying to do, but it could be tricky to get the ESP packets from the kernel through another tunnel (might be possible with some advanced routing/firewall tricks).

**Comment by zavataafnan** [comment]
Hi Tobian,
Thanks for answering, 
I have an OpenVPN as tun0 and I want to send all traffic of IPSEC or IKEv2 to this tunnel, So I changed the current tables from:

-A POSTROUTING -s 192.168.42.0/24 -o ens160 -j MASQUERADE
-A POSTROUTING -s 192.168.43.0/24 -o ens160 -j MASQUERADE
-A FORWARD -i ens160 -o ppp+ -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
-A FORWARD -i ppp+ -o ens160 -j ACCEPT
-A FORWARD -i ppp+ -o ppp+ -j ACCEPT
-A FORWARD -d 192.168.43.0/24 -i ens160 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
-A FORWARD -s 192.168.43.0/24 -o ens160 -j ACCEPT

to:

-A POSTROUTING -s 192.168.42.0/24 -o tun0 -j MASQUERADE
-A POSTROUTING -s 192.168.43.0/24 -o tun0  -j MASQUERADE
-A POSTROUTING -o tun0  -j MASQUERADE
-A FORWARD -i tun0 -o ppp+ -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
-A FORWARD -i ppp+ -o tun0 -j ACCEPT
-A FORWARD -i ppp+ -o ppp+ -j ACCEPT
-A FORWARD -d 192.168.43.0/24 -i tun0 -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
-A FORWARD -s 192.168.43.0/24 -o tun0 -j ACCEPT

But I run the traceroute, there is nothing going to the tun0. Thank you so much for any hint for me. 

Best Regards,
Mostafa

**Reply by tobiasbrunner** [reply]
Difficult to say from this as you haven't provided any information about the IPsec config and established connections (see #196).
